tools/go-cli/go-whisk-cli/commands/util.go

```diff
@@ -639,3 +639,22 @@ func unpackTar(inpath string) error {
     return nil
 }
 
+func checkArgs(args []string, minimumArgNumber int, maximumArgNumber int, commandName string, requiredArgMsg string) (*whisk.WskError) {
+    if len(args) < minimumArgNumber {
+        whisk.Debug(whisk.DbgError, fmt.Sprintf("%s command must have at least %d argument(s)\n", commandName, minimumArgNumber))
```

Comment:
Nit.  If `minimumArgNumber` == `maximumArgNumber`, then IMHO the trace msg would be clearer as `...command must have exactly %d argument(s)`. 

Maybe something like
```
var exactlyOrAtLeast string = "exactly"
var exactlyOrNoMoreThan string = "exactly"
if (minimumArgNumber != maximumArgNumber) {
    exactlyOrAtLeast = "at least"
    exactlyOrNoMoreThan = "no more than"
}
...fmt.Sprintf("%s command must have %s arguments(s)\n", commandName, exactlyOrAtLeast, minimumArgNumber))
```